packages/react/src/components/Dropdown/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import ReactSelect from 'react-select';
+import Icon from '../Icon';
+import UiText from '../UiText';
+import { safeInvoke } from '../../utils';
+
+const customStyles = {
+  control: () => ({
+    border: 0,
+    backgroundColor: 'transparent',
+    minHeight: 'initial',
+    display: 'flex',
+    justifyContent: 'space-between',
+  }),
+  indicatorSeparator: provided => ({
+    ...provided,
+    display: 'none',
+  }),
+  indicatorContainer: provided => ({
+    ...provided,
+    marginRight: '1rem',
```

Comment:
Use :flamingo: vars?

Reply:
That's another problem since it's the inline style, flamingo vars are in CSS variables, so cannot be used here.
We could use flamingo vars, if these all were in CSS with `!important`.
Both solutions stink a bit. Dunno. What do you thinkk?

Reply:
You should still be able to do `marginRight: 'var(...)',` 🤔